docs/wiki/Examples.md

```diff
@@ -50,3 +53,4 @@ Use the links below to explore these examples in more detail.
 [wiki_create_custom_policies_policy_sets_and_assignments]: %5BExamples%5D-Create-Custom-Policies-Policy-Sets-and-Assignments "Wiki - Create Custom Policies, Policy Sets and Assignments"
 [wiki_assign_a_built_in_policy]:                           %5BExamples%5D-Assign-a-Built-in-Policy "Wiki - Assign a Built-in Policy"
 [wiki_create_and_assign_custom_rbac_roles]:                %5BExamples%5D-Create-and-Assign-Custom-RBAC-Roles "Wiki - Create and Assign Custom RBAC Roles"
+[wiki_output_reference]:                                   %5BExamples%5D-Deploy-Output-Values "Wiki - Deploy Output Values"
```

Comment:
I'm not sure "Deploy" is the right verb for this guidance. Could we rename to something like `Use module outputs`?